payload/selfcheck/config.yaml

```diff
@@ -42,4 +52,14 @@ dataflow-problems:
         - type: "^int$" # not enough data in an int to do terminal injection
         - type: "^bool$" # same for bool
         - type: "^float" # same for floats
+  slicing:
+    # Check that the arguments of MustCompile are regexes defined entirely statically (i.e. all data flowing to the
```

Comment:
This isn't exactly true since the backtrace analysis won't report any errors if there are data flows to non-constants. We'd need to add extra functionality in the analysis to do this

Reply:
True, it's more like we check the results of the analysis to make sure that it's the case.